### PR TITLE
[minor] Fix JDBC doc after #2229

### PR DESCRIPTION
**Pull request by prabhjyotsingh**
### What is this PR for?
Fix JDBC doc after #2229.

### What type of PR is it?
[Bug Fix]

### Questions:
* Does the licenses files need update?
* Is there breaking changes for older versions?
* Does this needs documentation?

### CODE REVIEW

**Comment by prabhjyotsingh** [comment]
@AhyoungRyu  can you help review this PR. With that (#2229), I assumed all syntax of MD will work for documentation, but it didn't generate the table.

File: docs/interpreter/jdbc.md [hunk 1]
```diff
@@ -537,20 +537,40 @@ Here are some examples you can refer to. Including the below connectors, you can
 When Zeppelin server is running with authentication enabled, then the interpreter can utilize Hive's user proxy feature i.e. send extra parameter for creating and running a session ("hive.server2.proxy.user=": "${loggedInUser}"). This is particularly useful when multiple users are sharing a notebooks.
```
Comment (AhyoungRyu) [inline docs/interpreter/jdbc.md]: I think `are sharing a notebooks` should be `a notebook` :)

File: docs/interpreter/jdbc.md [hunk 2]
```diff
@@ -537,20 +537,40 @@ Here are some examples you can refer to. Including the below connectors, you can
...
-  *Properties*
 
-  | name                      | value                                                                                             |
-  |:------------------------- |:--------------------------------------------------------------------------------------------------|
-  | hive.driver               | org.apache.hive.jdbc.HiveDriver                                                                   |
-  | hive.password             |                                                                                                   |
-  | hive.url                  | jdbc:hive2://hive-server-host:2181/;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=hiveserver2 |
-  | hive.proxy.user.property  | hive.proxy.user.property                                                                          |
-  | zeppelin.jdbc.auth.type   | SIMPLE                                                                                            |
+  Example configuration
```
Comment (AhyoungRyu) [inline docs/interpreter/jdbc.md]: Then do we need this line?

File: docs/interpreter/jdbc.md [hunk 3]
```diff
@@ -537,20 +537,40 @@ Here are some examples you can refer to. Including the below connectors, you can
...
 
+##### Properties
```
Comment (AhyoungRyu) [inline docs/interpreter/jdbc.md]: Yeah you need to use static table tag to generate the table. Seems it's working well now. 
screen shot 2017-05-04 at 11 21 13 am

File: docs/interpreter/jdbc.md (hunk 2 above)
Comment (prabhjyotsingh) [inline docs/interpreter/jdbc.md]: Yes, I was thinking of sample/example configuration, or should I change the line below `##### Properties` to `##### Sample configuration` ?

Comment (AhyoungRyu) [inline docs/interpreter/jdbc.md]: Yeah that would be better i think :)

**Comment by prabhjyotsingh** [comment]
@AhyoungRyu  Thank you for the review, have made the recommended changes.

**Comment by felixcheung** [comment]
ah sorry I should have picked this up in review

**Comment by prabhjyotsingh** [comment]
Thank you @AhyoungRyu, @felixcheung for the review.
Will merge this to master, if no more discussion.

**Pull request closed without merging**